### ISSUE TITLE
Follow best a11y practices for the password strength bubble.

### ISSUE THREAD

**Issue by shane-tomlinson** [mozilla/fxa-content-server#6299]
Marco Zehe reminded me to keep accessibility in mind for the new password strength bubble. Showing the checkboxes as background images is not screen reader friendly.

**Comment by shane-tomlinson** [comment]
@MarcoZehe was able to point out several areas the strength balloon can be improved:

* Do not autofocus the password field when invalid, it makes it to that screen reader users cannot easily read the balloon.
* The indicators whether a criteria is met or fails is purely visual. Give each criterion an id, and then when the password input is invalid, set an `aria-described-by` on the input element.
* When the input is invalid due to one of the criterion in the balloon, set an `aria-invalid="true"` attribute on the input element. The "pattern" attribute catches passwords that are too short, but not passwords that are too common or too similar to the email address.

There were other issues we discussed outside of this, I'll open separate issues for those.

**Comment by SorinaFlorean** [comment]
Verified on Stage: Train 116.
